firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -3910,6 +3910,7 @@ extension BrowserViewController: TabManagerDelegate {
                                               title: tab.lastTitle,
                                               lastExecutedTime: tab.lastExecutedTime)
         }
+        urlBar.updateProgressBar(Float(0))
```

Comment:
since it's not reproducible for the new tool bar, should we add a flag here so that when we move to the new toolbar code, we know to remove this as well? 

Or maybe adding a comment would be great for context in the future!